live/src/agama-installer.kiwi

```diff
@@ -134,53 +133,72 @@
         <package name="blog" arch="s390x" />
         <package name="libblogger2" arch="s390x" />
         <package name="xauth"/>
-        <package name="patterns-base-x11"/>
-        <namedCollection name="x11"/>
         <package name="patterns-yast-yast2_basis"/>
         <package name="MozillaFirefox" arch="aarch64,x86_64"/>
         <package name="MozillaFirefox-branding-openSUSE" arch="aarch64,x86_64"/>
         <package name="libpwquality-tools"/>
         <package name="NetworkManager"/>
-        <package name="ruby3.3-rubygem-byebug"/>
-        <package name="ruby3.3-rubygem-agama-yast"/>
         <package name="agama"/>
         <package name="agama-web-ui"/>
         <package name="agama-cli"/>
         <package name="agama-auto"/>
         <package name="microos-tools"/>
-        <package name="staging-build-key"/>
         <package name="icewm-lite"/>
         <package name="xinit"/>
+        <package name="xorg-x11-server"/>
+        <package name="xorg-x11-fonts-core"/>
+        <package name="xf86-input-libinput" />
         <package name="psmisc"/>
         <package name="joe"/>
         <package name="checkmedia"/>
         <package name="spice-vdagent"/>
         <package name="libtss2-tcti-device0"/>
         <package name="jq"/>
+        <package name="yast2-schema"/>
         <archive name="root.tar.xz"/>
     </packages>
     <!-- additional packages for the openSUSE flavors -->
     <packages type="image" profiles="openSUSE,openSUSE-Playwright">
-      <package name="agama-products-opensuse"/>
-      <package name="openSUSE-build-key"/>
+        <package name="agama-products-opensuse"/>
+        <package name="grub2-branding-openSUSE" arch="aarch64,x86_64"/>
+        <package name="patterns-openSUSE-base"/>
+        <package name="ruby3.3-rubygem-agama-yast"/>
```

Comment:
This `ruby3.3` vs. `ruby3.2` packages looks ugly, but there is probably no better way... :worried:

Reply:
Yes, I agree. Quoting @jreidinger:

> I think plan for SLE16 is to have suseruby instead of ruby3.3 and it will mean ruby that is supported only for suse products